src/state/contexts/ModalState.tsx

```diff
@@ -5,6 +5,7 @@ import { ModalAction } from '../reducers/ModalReducer';
 
 export type ModalState = {
   showUpgradeAntModal: boolean;
+  showEditUndernameModal: boolean;
```

Comment:
Not for this PR, but we should discuss the way modal's are done in Network Portal and see about a similar pattern for this app in the future.